release-notes/8.0/supported-os.json

```diff
@@ -207,6 +217,12 @@
                         "9",
                         "8"
                     ],
+                    "exceptional-versions": [
+                        {
+                            "version": "7",
+                            "note": "Red Hat Enterprise Linux 7 was explicitly never supported."
+                        }
+                    ],
```

Comment:
@richlander Maybe we want to give a better explanation here.

Reply:
It was explicitly not supported due to glibc compatibility. When we moved from CentOS 7 to Ubuntu 16.04 as the compilation target, we dropped RHEL 7, CentOS 7, and any other distro in that version family.

Context: https://github.com/dotnet/runtime/issues/83428